### ISSUE TITLE
CommandButton: Incorrect State OFF defined

### ISSUE THREAD

**Issue by ericajlin** [OfficeDev/office-ui-fabric-react#4951]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__:
 OS: Win 10 RS3 1709
Build: OS Build 16299.248
Platform: Edge + Narrator

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Normal) 

Products/sites affected: (if applicable)

#### Describe the issue:
Incorrect state 'OFF' defined for Command buttons
Narrator is always reading 'OFF *Button Text*' even when the button is enabled.
Issue repo's in Edge browser using Narrator.

#### Actual behavior:
Button controls are not accessible as narrator is reading "Off ..." for non-disabled buttons in Edge browser. 

#### Expected behavior:
Button should only read off when it is disabled.

**Comment by JasonGore** [comment]
Confirmed as issue. The problem is that `aria-pressed` attribute is always provided for buttons, but should only be provided for buttons that are toggle switches with distinct on and off states. This is why Narrator reads "Off" for all buttons (and conversely, if `Mark as checked` is selected [here](https://developer.microsoft.com/en-us/fabric#/components/button) then Narrator reads "ON" for all buttons.)

Exacerbating the issue is that `checked` attribute applies to all buttons, not just those that are toggle buttons. There is no existing functionality to indicate a button is a `toggle` button so there isn't a clearcut way of implementing `aria-pressed` in a way that communicates consumer intent.
